src/xdsl/dialects/std.py

```diff
@@ -29,15 +29,15 @@ def __post_init__(self):
 
         self.ctx.register_op(Cmpi)
 
-        self.f32 = Float32Type.get()
-        self.i64 = IntegerType.get(64)
-        self.i32 = IntegerType.get(32)
-        self.i1 = IntegerType.get(1)
+        self.f32 = Float32Type()
+        self.i64 = IntegerType.from_width(64)
+        self.i32 = IntegerType.from_width(32)
+        self.i1 = IntegerType.from_width(1)
 
     # TODO make this generic in the type
     def constant(self, val: int, typ: Attribute) -> Operation:
-        return Constant.create([], [typ],
-                               attributes={"value": IntegerAttr.get(val, typ)})
+        return Constant.create(
+            [], [typ], attributes={"value": IntegerAttr.build(val, typ)})
```

Comment:
Why do we use `build` here, but a specific builder (`from_int_and_with`) later on?

Reply:
Yes, I'll change it back to `from_int_and_width`